run_dir/design/flowcells.html

```diff
@@ -110,7 +110,7 @@ <h1><span id="page_title">Flowcells</span>
                      {% if 'lane_info' in flowcells[onefc] %}
                          {% for onelane in flowcells[onefc]['lane_info'] %}
                              <samp style="max-width:50px;">{{ flowcells[onefc]['lane_info'][onelane].get('q30_bases', 'NA') }}</samp>
-                             {% set run_setup = flowcells[onefc].get('recipe', '').split('x')[-1] %}
+                             {% set run_setup = flowcells[onefc].get('longer_read_length', '') %}
```

Comment:
Will the run_setup be `longer_read_length`too? Isn't `longer_read_length` an integer?